landing_page/static/landing_page/css/movie.css

```diff
@@ -3,36 +3,123 @@
 #movie-button {
     font-size: 28.7px;
     font-family: var(--primary-font);
+    font-weight: 600;
     color: var(--black);
     border-radius: 12px;
     text-align: center;
     padding: 10px;
-
+    border: none;
 }
 
 #movie_summary {
     font-family: var(--primary-font);
     font-size: 32px;
+    text-align: left;
+    border: 1px solid #e1e1e1;
+    border-radius: 5px;
+    padding: 20px;
+    width: 100%;
+}
+
+#streaming_info {
+    margin-top: 15px;
 }
+
 #movie_name {
-    font-size: 32px;
+    font-size: 40px;
     font-family: var(--primary-font);
     margin-left: 10px;
-    display: flex;
-    
+    display: inline;
+    padding: 10px;
+    text-align: center;
 }
 
 #movie_button_nav {
     margin-left: 0px;
+    margin-bottom: 10px;
+    margin-top: 5px;
+}
+
+#movie-summary p {
+    margin-left: 1rem;
+    margin-right: 1rem;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+}
+
+.content_container img {
+    height: 600px;
+    width: auto;
+}
+
+/*TODO mobile styling for tooltip*/ 
+/* tooltip modified & repurposed from landing_page/index.css */
+
+.tooltip_button {
```

Comment:
i too used the tooltip stuff for the recommendations page - maybe we should throw these into like a 'tooltip.css'...
we can look about doing that later though! zero worry right now.